src/pages/Error.tsx

```diff
@@ -0,0 +1,93 @@
+import SadIcon from "@mui/icons-material/SentimentVeryDissatisfied";
+import SadlyIcon from "@mui/icons-material/SentimentDissatisfied";
+import { Typography, Box, Button, useMediaQuery } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import { useNavigate } from "react-router-dom";
+import { useState, useEffect } from "react";
+
+export default function ErrorPage() {
+  const theme = useTheme();
+  const navigate = useNavigate();
+  const matches = useMediaQuery(theme.breakpoints.down("sm"));
+
+  const [icon, setIcon] = useState(true);
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      setIcon((prevIcon) => !prevIcon);
+    }, 3000);
```

Comment:
interesting solution! :) 

please add `// TODO update animation` comment for update this in the future